### ISSUE TITLE
How can I fit my hand image to manopth model

### ISSUE THREAD

**Issue by momo1986** [hassony2/manopth#5]
Hello, @hassony2 .

Thanks for sharing.

I have tried the demo.

If I have my own hand image, how can I fit it into the model?

In other words, looks like pose vector and shape vector is an essential input, how can I get these two tensors from own data(e.g., RGB data)?

Thanks & regards!

**Comment by MLsmaller** [comment]
> Hello, @hassony2 .
> 
> Thanks for sharing.
> 
> I have tried the demo.
> 
> If I have my own hand image, how can I fit it into the model?
> 
> In other words, looks like pose vector and shape vector is an essential input, how can I get these two tensors from own data(e.g., RGB data)?
> 
> Thanks & regards!



> Hello, @hassony2 .
> 
> Thanks for sharing.
> 
> I have tried the demo.
> 
> If I have my own hand image, how can I fit it into the model?
> 
> In other words, looks like pose vector and shape vector is an essential input, how can I get these two tensors from own data(e.g., RGB data)?
> 
> Thanks & regards!
That's the question I want to ask！

**Comment by MLsmaller** [comment]
This is also my problem!

**Comment by hassony2** [comment]
Mano pose and shape parameters can be estimated from RGB data using a neural network.

I provide some code to perform RGB-image to mano estimation at [obman_train](https://github.com/hassony2/obman_train) which is the code I used for our _Learning joint reconstruction of hands and manipulated objects_ paper.

You might also be interested by [HAMR](https://github.com/MandyMo/HAMR) by [MandyMo](https://github.com/MandyMo).

Best,

Yana

**Comment by MedlarTea** [comment]
Hello, @hassony2 . Thanks for sharing! 
I'm a novice. I have a problem about downloading MANO model.
When i input "wget  --no-check-certificate https://psfiles.is.tuebingen.mpg.de/downloads/mano/mano_v1_2-zip", it returned a text--"Permission denied."
Thanks!!!

**Comment by dimtziwnas** [comment]
Hi @MedlarTea , for setting up the pipeline please follow the exact instructions of @hassony2  in the read_me file.

The MANO model is hosted on our MPG servers, and users have to create an account explicitly agree with our license before downloading the model, using their name and email.  This means that the URL that you are using cannot be simply passed to other users and used in a wget command similar to what you try.  This is a fake URL that works only when you are signed in on the MANO website, our web application takes care of this. 

Bottom line, please follow step-by-step the exact instructions that @hassony2 created :)

**Comment by MedlarTea** [comment]
Oh! Thanks a lot!!! @dimtziwnas

**Comment by MedlarTea** [comment]
Hi @dimtziwnas , Thanks for your reply again!!! I have an another problem. In "https://github.com/hassony2/obman_train", do you know why I can't get a reply about the ObMan dataset from the author @hassony2 ? I have sent an e-mail for requesting the dataset almost two weaks. 
Thanks a lot!!!

**Comment by hassony2** [comment]
@MedlarTea  could you [request](https://www.di.ens.fr/willow/research/obman/data/) the dataset again ? There might have been a problem with the e-mail address you provided or on my side !

Best !

Yana

**Comment by MedlarTea** [comment]
@hassony2 , I have requested again! My mailbox is QQ mailbox, and I have received your automated message again~

Best wishes!

Ye
